eth/blockwatch/block_watcher.go

```diff
@@ -146,6 +146,25 @@ func (w *Watcher) InspectRetainedBlocks() ([]*MiniHeader, error) {
 	return w.stack.Inspect()
 }
 
+func (w *Watcher) syncToLatestBlock() error {
+	lastBlock, err := w.client.HeaderByNumber(nil)
+	if err != nil {
+		return err
+	}
+
+	for {
+		latestHeader, err := w.stack.Peek()
+		if err != nil {
+			return err
+		}
+		if lastBlock.Number.Cmp(latestHeader.Number) == 0 {
+			break
+		}
+		w.pollNextBlock()
```

Comment:
Let's handle the error returned here.

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/1729/commits/511c85802c591398ed2ba931af5013a70cea502a